src/lib/github-api.js

```diff
@@ -195,6 +187,17 @@ async function getTargetBranch(targetBase, repoName, octokit) {
   return repo.data?.default_branch || targetBase;
 }
 
+async function getIssueTitle(branchName, octokit) {
+  const infos = getReposAndIssuesId(branchName);
+  if (infos.length === 0) {
+    return branchName;
+  }
```

Comment:
Can we please keep the branch name if there is more than 1 issue found? Otherwise, I will have to change the title every time...

Reply:
For me the title of a PR should never be the branch name. It's too cryptic and half of the branch name is useless for the reviewers.

I think that using the first issue title for the PR title is good, and we could add `(and <n> more tasks)` at the end of the title if we want to convey that there are other things too.

What do you think?

Reply:
I'll change that! ;-)